prototype2/CMakeLists.txt

```diff
@@ -72,7 +63,7 @@ add_subdirectory(udp)
 add_subdirectory(adc_readout)
 
 add_custom_target(runefu
-    COMMAND efu "-d" "../lib/mgmesytec" "-s" "1"
+    COMMAND efu "-d" "../bin/modules/mgmesytec" "-s" "1"
```

Comment:
Seeing as the previous version of the `runefu` target is not that intelligent, I am not going to force you to improve it. I want to mention however that it is not working when testing it because it makes (sometimes) faulty assumptions about where files are. This is easily fixed.